packages/journey-manager/src/index.ts

```diff
@@ -87,6 +116,22 @@ const matchesUrlCondition = (urlCondition: UrlCondition): boolean => {
 export const run = (config: Config, triggers: Triggers): (() => void) => {
   const client = create(config);
 
+  const triggeredSteps = getTriggeredSteps(config.conversationId);
+
+  const sendStep = (stepId: string, once: boolean): void => {
+    if (triggeredSteps.includes(stepId)) {
+      triggeredSteps.push(stepId);
+      saveTriggeredSteps(config.conversationId, triggeredSteps);
+      if (once) {
+        return;
+      }
+    }
```

Comment:
Something doesn't seem quite right here. How would a stepId ever get into `triggeredSteps` in the first place? Perhaps a `!` missing in the conditional?

Reply:
You're totally right, updated the logic!